### PR TITLE
Add awaitingDcnProcessing field to ExceptionRecord

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-814

### Change description ###
- Add `awaitingDcnProcessing` field to ExceptionRecord
- Set `awaitingDcnProcessing` value as "Yes" when the envelope contains payments.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/ExceptionRecord.java [hunk 1]
```diff
@@ -39,6 +39,10 @@
     // Yes/No field indicating if there are warnings to show
     public final String displayWarnings;
 
+    // Yes/No field indicating if the payments are processed
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/ExceptionRecord.java]: indicating if the payment document control numbers are processed(payment processed is a different thing :) )

File: src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/ExceptionRecordMapperTest.java [hunk 2]
```diff
@@ -129,6 +131,35 @@ public void mapEnvelope_sets_envelope_id_to_null_when_jurisdiction_does_not_supp
         assertThat(exceptionRecord.envelopeId).isNull();
     }
 
+    @Test
+    public void mapEnvelope_sets_awaitingDcnProcessing_to_no_when_envelope_contains_payments() {
```
Comment (nitinprabhu) [inline src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/ExceptionRecordMapperTest.java]: doesn't match with assertion .You mean `mapEnvelope_sets_awaitingDcnProcessing_to_yes_when_envelope_contains_payments` ?

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/ExceptionRecord.java (hunk 1 above)
Comment (aliveni) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/ExceptionRecord.java]: updated

File: src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/ExceptionRecordMapperTest.java (hunk 2 above)
Comment (aliveni) [reply src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/ExceptionRecordMapperTest.java]: renamed the test

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/ExceptionRecord.java [hunk 3]
```diff
@@ -39,6 +39,10 @@
     // Yes/No field indicating if there are warnings to show
     public final String displayWarnings;
 
+    // Yes/No field indicating if the payment document control numbers are processed
+    @JsonProperty("awaitingDCNProcessing")
+    public final String awaitingDcnProcessing;
```
Comment (lgonczar) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/ExceptionRecord.java]: In current form this PR can only be merged once all services have this field added to their exception record definitions in aat and prod.

Comment (aliveni) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/ExceptionRecord.java]: Yeah. As per the ticket, I'll have to merge this PR to demo branch first. After QA, I'll merge this pr to master

**Pull request merged**